src/machine/machine-dbus.c

```diff
@@ -1006,7 +1006,7 @@ int bus_machine_method_bind_mount(sd_bus_message *message, void *userdata, sd_bu
                 }
 
                 mount_inside = strjoina("/run/host/incoming/", basename(mount_outside));
-                r = mount_nofollow_verbose(LOG_ERR, mount_inside, dest, NULL, MS_MOVE, NULL);
+                r = mount_follow_verbose(LOG_ERR, mount_inside, dest, NULL, MS_MOVE, NULL);
```

Comment:
I'm not sure I understand, sorry - we use nofollow and resolve symlinks because it's a well-known source of attacks and jail escape, given the kernel follows them. In your example you are binding /mnt - why is that a symlink? To where?